editor/editor_node.cpp

```diff
@@ -2461,6 +2461,19 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			OS::get_singleton()->set_window_fullscreen(!OS::get_singleton()->is_window_fullscreen());
 
 		} break;
+		case EDITOR_SCREENSHOT: {
+			Viewport *viewport = editor_interface->get_editor_viewport()->get_viewport();
+			viewport->set_clear_mode(Viewport::CLEAR_MODE_ONLY_NEXT_FRAME);
+			Ref<Image> img = viewport->get_texture()->get_data();
+			img->flip_y();
+			viewport->set_clear_mode(Viewport::CLEAR_MODE_ALWAYS);
+			Dictionary datetime = _OS::get_singleton()->get_datetime(false);
+			for (int32_t i = 0; i < datetime.size(); i++) {
+				datetime[datetime.get_key_at_index(i)] = String(datetime.get_value_at_index(i)).pad_zeros(2);
+			}
+			Error error = img->save_png(String("user://" + TTR("editor") + "_{year}_{month}_{day}_{hour}_{minute}_{second}.png").format(datetime));
```

Comment:
I wouldn't make the "editor" part translatable, otherwise we should also make the date format localizable, and I don't see much value doing so for such a feature.
For the date, I'd suggest to simplify to "{year}{month}{day}_{hour}{minute}{second}".